packages/proxy-agent/package.json

```
@@ -69,7 +69,7 @@
   "dependencies": {
     "@web5/agent": "0.2.4",
     "@web5/common": "0.2.1",
-    "@web5/crypto": "0.2.2",
+    "@web5/crypto": "0.2.3",
```

`@web5/proxy-agent` needs to be updated before using `@web5/crypto` version `0.2.3` so reverting.
Details here: https://github.com/TBD54566975/web5-js/pull/318
```suggestion
    "@web5/crypto": "0.2.2",
```